### DISCUSSION TITLE
Reuse gRPC channels

### DISCUSSION THREAD

**Discussion by OmGuptaIND** [connectrpc/connect-es#779] [Q&A]
Do we have this feature in the connect-es 
My requirement is simple will it be able to handle multiple requests, possibly more than 10k+, to different microservices 
gRPC implements something called channels to manage on a single connection to a service

Couldn't find anything relevant on the docs

For Ref: 
https://learn.microsoft.com/en-us/aspnet/core/grpc/performance?view=aspnetcore-7.0#reuse-grpc-channels

**Comment by srikrsna-buf** [comment]
It can handle multiple requests on a single connection in fact that is the only way it does it today: https://github.com/connectrpc/connect-es/blob/main/packages/connect-node/src/http2-session-manager.ts#L215
